components/movementsensor/rtkutils/gpsrtk.go

```diff
@@ -1,290 +1,121 @@
-// Package rtkutils defines a gps and an rtk correction source
-// which sends rtcm data to a child gps
-// This is an Experimental package
+// Package rtkutils implements functions that is used in gpsrtkserial and
+// gpsrtkpmtk.
 package rtkutils
 
 import (
-	"context"
+	"bufio"
+	"encoding/base64"
 	"errors"
+	"fmt"
 	"io"
-	"math"
+	"net"
+	"strings"
 	"sync"
 
-	"github.com/golang/geo/r3"
-	geo "github.com/kellydunn/golang-geo"
+	"github.com/de-bkg/gognss/pkg/ntrip"
 
-	"go.viam.com/rdk/components/board"
-	"go.viam.com/rdk/components/movementsensor"
-	gpsnmea "go.viam.com/rdk/components/movementsensor/gpsnmea"
 	"go.viam.com/rdk/logging"
-	"go.viam.com/rdk/resource"
-	"go.viam.com/rdk/spatialmath"
 )
 
-// A RTKMovementSensor is an NMEA MovementSensor model that can intake RTK correction data.
-type RTKMovementSensor struct {
```

Comment:
This struct is not moved, it's just gone. Any chance you put it in a new file but forgot to add the file to the PR? If not, what did this struct do and why don't we need it any more?